src/2023/index.html

```diff
@@ -13,6 +13,29 @@
           <h1 class="main-title">Helvetic Ruby 2023</h1>
           <p class="lead">A Ruby conference in Switzerland.</p>
           <p class="venue-date"><time>24.11.2023</time> in Bern, Switzerland</p>
+
+          <div class="social">
+            <div class="social--networks">
+              <a href="https://ruby.social/@helvetic_ruby" title="Follow us on Mastodon">
+                <div class="social--network-icon">
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" fill="currentColor" stroke="currentColor">
+                    <!--! Font Awesome Pro 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
```

Comment:
Are *you* paying the licensing fees?

Reply:
Thanks for that question. The icons are in the _FREE_ set of Font Awesome and the trademark policies of Twitter and Mastodon allow the use (it would be weird if not).

Although the Twitter policy says we may only use the logo in white or blue, nothing else. Here we use it white with 50% opacity, and white on hover... I don't know how tolerable that is, what do you think. 🙃

Reply:
May be we should/could/have to remove the comment?

Reply:
Do Mastodon and Twitter provide SVG icons we can use directly without taking them from Font Awesome? I'd try that way.

Reply:
Good idea! I replaced the FA icon with the offical ones.